### ISSUE TITLE
PSA 119 - Alpha 'headers'

### ISSUE THREAD

**Issue by jag3773** [unfoldingWord-dev/ts-desktop#977]
_From @ChrisJarka on December 21, 2016 14:57_

Jesse - 
In tS  v11 the Alphabet headers/markers are not displaying correctly in PSA 119.

![psa 119](https://cloud.githubusercontent.com/assets/9850185/21393507/c0176664-c762-11e6-9f44-63f40f0f706f.png)

The Hebrew alphabet headers are marked as a \d in the USFM. The \d is before the verse marker. Hence why the ALPHA shows up correctly (there is no verse before) but the BETH shows up before the verse 9 marker on verse 8. Here is a truncation of what is in the USFM;

\p 
6 \d ALEPH. 
7 \b 
8 \q 
9 \v 1 Blessed are those whose ways are blameless, 
10 \q who walk in the law of Yahweh. 
.....
33 \v 7 I will give thanks to you with an upright heart 
34 \q when I learn your righteous decrees. 
35 \q 
36 \v 8 I will observe your statutes; 
37 \q do not leave me alone. 
38 \b 
39 
 
40 \s5 
41 \d BETH. 
42 \q 
43 \v 9 How can a young person keep his path pure? 
44 \q By obeying your word. 
45 \q 
46 \v 10 With my whole heart I seek you; 
47 \q Do not let me stray from your commandments. 

_Copied from original issue: Door43/ulb-en#401_

**Comment by jag3773** [comment]
cc @cckozie @neutrinog @klappy

**Comment by neutrinog** [comment]
That's because the content is chunked by verse. So BETH appears on the previous chunk.

**Comment by ChrisJarka** [comment]
@neutrinog exactly. I'm wondering if it is a formatting or a rendering change that needs to be made so that the alpha shows on the correct verse.
Or are the alpha notations not inspired and are not necessary. (Stepping in deep here.)
